doc/api/debugger.md

```diff
@@ -202,3 +202,35 @@ To start debugging, open the following URL in Chrome:
 ```
 
 [TCP-based protocol]: https://github.com/v8/v8/wiki/Debugging-Protocol
+
+## Trace Event Support for Node.js
+
+Trace Event provides a mechanism to centralize tracing information generated by
+V8, Node core, and userspace code.
+
+Tracing can be enabled by passing the `--enable-tracing` flag when starting a
+Node.js application.
+
+Tracing can be configured by additionally specifying a configuration file using the
+`--trace-config` flag:
+
+```txt
+node --enable-trace --trace-config=trace-config.json server.js
+```
+
+where `trace-config.json` may override any of the default configuration values:
```

Comment:
More clarification would be helpful here:
 * Are the values below actually the defaults, or just examples?
 * What are the valid values for `record_mode`?
 * What do each of those boolean flags mean?
 * Are there any other known categories for the node/v8 engine? (I realize user code can add arbitrary tracing categories.)